### PR TITLE
replacing conduit hdf5 API calls with the more generic conduit::relay…

### PR DESCRIPTION
**Pull request by davidHysom**
…::io::IOHandle

### CODE REVIEW

**Comment by davidHysom** [comment]
closing this, since the templated sample_list that is under development makes this unnecessary

**Comment by bvanessen** [comment]
I think that we still want to use this PR because it should provide a nicer method than more explicit templating.  Are these changes pushed into the template refactor PR?

Brian C. Van Essen
vanessen1@llnl.gov
(w) 925-422-9300
(c) 925-290-5470

**Comment by davidHysom** [comment]
OK, I've reopened.
>Are these changes pushed into the template refactor PR?
Jae-Seung can answer definatively. I sent him info on hdf5 calls and their equivlents with the new IOHandle class, so my guess is 'yes.'
________________________________

**Comment by JaeseungYeom** [comment]
No, it has not been integrated into the PR for template version of sample list. I have been attending DOE workflow workshop/hackathon at Sandia. I will work on that when I am back.

**Comment by davidHysom** [comment]
I'm in favor of renaming sample_list_jag to sample_list,
since it will be used for other data sources.
Any objections?

(leaving soon for a medical appt; back in a couple of hours)
________________________________

**Comment by bvanessen** [comment]
Yes, we should rename it.

Brian C. Van Essen
vanessen1@llnl.gov
(w) 925-422-9300
(c) 925-290-5470

**Comment by JaeseungYeom** [comment]
Most of the renaming is already done in the template version of sample list

**Comment by davidHysom** [comment]
Brian: in light of Jae-Seung's comment, would you like me to continue working on my PR, or re-close it?
________________________________

**Comment by bvanessen** [comment]
We don’t need to modify PR923 to rename the files.  We can take care of that in the templating part of it.

Brian C. Van Essen
vanessen1@llnl.gov
(w) 925-422-9300
(c) 925-290-5470

**Comment by davidHysom** [comment]
So ... I should continue working on 923?
________________________________

File: include/lbann/data_readers/sample_list_jag_impl.hpp [hunk 1]
```diff
@@ -147,25 +147,25 @@ inline sample_list_header sample_list_jag::read_header(std::istream& istrm, cons
...
 
 inline hid_t sample_list_jag::get_conduit_bundle_samples(std::string conduit_file_path, std::vector<std::string>& sample_names, size_t included_samples, size_t excluded_samples) {
-  hid_t hdf5_file_hnd = 0;
+  io_t file_hnd = 0;
   bool retry = false;
   int retry_cnt = 0;
   do {
     try {
-      hdf5_file_hnd = conduit::relay::io::hdf5_open_file_for_read( conduit_file_path );
+      file_hnd.open( conduit_file_path );
```
Comment (bvanessen) [inline include/lbann/data_readers/sample_list_jag_impl.hpp]: Where does the code specify that it is manipulating an HDF5 file?

Comment (davidHysom) [reply include/lbann/data_readers/sample_list_jag_impl.hpp]: This PR isn't actually ready for review. I'm unsure what you're asking. If you're referring to the added line, it's incomplete; should read file_hnd.open( conduit_file_path, "hdf5");

**Pull request closed without merging**